modules/framework/galasa-parent/dev.galasa.framework.api.common/src/main/java/dev/galasa/framework/api/common/ServletErrorMessage.java

```diff
@@ -133,6 +133,7 @@ public enum ServletErrorMessage {
     GAL5086_FAILED_TO_GET_DEFAULT_ROLE                (5086, "E: Failed to retrieve a default role id for a user with no role set."),
     GAL5087_BAD_USER_UPDATE_FIELD_ROLE                (5087, "E: Server detected an invalid 'role' field value from a client wishing to update a user record. The role field must be less than 128 characters and a consist of alphanumeric characters, '-' (hyphen) or '_' (underscore)"),
     GAL5088_FORBIDDEN_USER_DELETE_THEMSELVES          (5088, "E: It is not permitted for a user to delete their own user record. Ask another Galasa service administrator to delete your user record for you."),
+    GAL5089_FORBIDDEN_USER_DELETE_SERVICE_OWNER       (5089, "E: It is not permitted for a user to delete an owner of the Galasa service. Ask a Galasa service administrator to change the list of owners in the kubernetes configuration."),
```

Comment:
Maybe this is my lack of understanding as I've not been super involved in the RBAC stuff but reading this I'm confused what the different is between owners and service administrators, and who has the most privilege, etc?